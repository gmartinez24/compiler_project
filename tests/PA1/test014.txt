// Brendan Fattig
// testing nested comparisons

main

function isOdd (int n) : bool {
    return (n % 2 == 0) == true;
}