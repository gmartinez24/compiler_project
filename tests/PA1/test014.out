MAIN
FUNC
IDENT isOdd
OPEN_PAREN
INT
IDENT	n
CLOSE_PAREN
COLON
BOOL
OPEN_BRACE
RETURN
OPEN_PAREN
IDENT   n
MOD
INT_VAL 2
EQUAL_TO
INT_VAL 0
CLOSE_PAREN
EQUAL_TO
TRUE
SEMICOLON
CLOSE_BRACE
EOF